### PR TITLE
[webgl] Support PRELU and LOGISTIC ops

### PR DESCRIPTION
**Pull request by cuiyanx**
1. Add PRELU ops on webgl.
2. Realize L2_NORMALIZATION and LOGISTIC ops on webgl.

Signed-off-by: cuiyan <yanx.cui@intel.com>

### CODE REVIEW

**Comment by cuiyanx** [comment]
@ibelem  @BruceDai please review

File: src/nn/Enums.js [hunk 1]
```diff
@@ -1264,6 +1264,42 @@ export const OperationCode = {
    */
   MAXIMUM: 65,
 
+  /** Computes parametric rectified linear activation on the input tensor element-wise.
```
Comment (BruceDai) [inline src/nn/Enums.js]: @cuiyanx Please align with the explanation of [ANEURALNETWORKS_PRELU](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a82e7e7c709ad117a0178203217be30cf), update "**Computes parametric rectified linear activation on the input tensor element-wise.** "with "**_Parametric Rectified Linear Unit._**" 
Also using "**_It follows: f(x) = alpha * x for x < 0, f(x) = x for x >= 0, where alpha is a learned array with the same OperandCode and compatible dimensions as input x._**"

File: src/nn/Enums.js [hunk 2]
```diff
@@ -1264,6 +1264,42 @@ export const OperationCode = {
...
+   * Supported tensor rank: from 1.
+   *
+   * Inputs:
+   * * 0: A tensor, specifying the input.
+   * * 1: A tensor of the same OperandCode, and compatible dimensions as input0,
+   *      specifying the alpha.
+   *
+   * Outputs:
+   * * 0: A tensor of the same OperandCode as input0. For a
+   *      ANEURALNETWORKS_TENSOR_QUANT8_ASYMM tensor, the scale and zeroPoint
```
Comment (BruceDai) [inline src/nn/Enums.js]: @cuiyanx Please update "**ANEURALNETWORKS_TENSOR_QUANT8_ASYMM**" with "**_TENSOR_QUANT8_ASYMM_**"

File: src/nn/Enums.js [hunk 3]
```diff
@@ -1264,6 +1264,40 @@ export const OperationCode = {
    */
   MAXIMUM: 65,
 
+  /** Parametric Rectified Linear Unit.
```
Comment (BruceDai) [inline src/nn/Enums.js]: @huningxin This explanation of PRELU is from [ANEURALNETWORKS_PRELU](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a82e7e7c709ad117a0178203217be30cf), please take a review, thanks

Comment (huningxin) [reply src/nn/Enums.js]: Looks good. Thanks.

**Comment by huningxin** [comment]
Looks good. Thanks @cuiyanx .

**Pull request merged**